docs/HACKING.md

```diff
@@ -3,15 +3,22 @@
 # Hacking on the WMCO
 
 ## Build
-To build the operator image, execute:
+To build and push the operator image, execute:
 ```shell script
-podman build . -t quay.io/<insert username>/wmco:$VERSION_TAG -f build/Dockerfile
+export OPERATOR_IMAGE=quay.io/<insert username>/wmco:$VERSION_TAG
```

Comment:
Since `$VERSION_TAG` is a user defined field like `<insert username>`, I feel it should be written as `<version tag>`

Reply:
I am not introducing the `$VERSION_TAG` in this PR but is existing usage. Changes like this warrants a separate PR given we are using `$*_TAG` in multiple places and should be changed across the board.